continuous_integration/environment-3.10-jdk11-dev.yaml

```diff
@@ -4,7 +4,7 @@ channels:
 - nodefaults
 dependencies:
 - adagio>=0.2.3
-- antlr4-python3-runtime>=4.9.2
+- antlr4-python3-runtime>=4.9.2, <4.10.0 # Max pin until https://github.com/fugue-project/fugue/issues/318 is resolved
```

Comment:
Looks like https://github.com/fugue-project/fugue/issues/318 should have been resolved with fugue 0.6.6; maybe we should swap out this comment to say that the problem occurs with `fugue<0.6.6`?

Reply:
Agreed, I'll update the comment but just as a quick note since we get fugue from `pip` the pip dependencies don't really impact what the conda solver picks up, so we'd need to keep this pin around even if we bump our fugue version to 0.6.6